src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final Random RANDOM = new Random();
+    private static final int MAX_BALL_NUMBER = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int ballNumber = RANDOM.nextInt(MAX_BALL_NUMBER);
+        String ballColor = colorSupplier.getRandomColor().name();
```

Comment:
pls pay attention to my [comment](again) :)

you forgot to call `toLowerCase()` - you need `green` but not `GREEN`